apps/api/src/app/domains/mentors/mentors.controller.ts

```diff
@@ -8,8 +9,8 @@ export class MentorsController {
   ) {}
 
   @Post()
-  getAll(){
-    console.log(1)
+  async createMentor(@Body() test: TestTemplate) {
+    console.log('controller : ' + test);
```

Comment:
change test variables to logic names ex.: payload and TestTemplate convert to interface